Orange/tests/test_evaluation_scoring.py

```diff
@@ -65,7 +65,7 @@ def test_call(self):
 
     def test_bayes(self):
         x = np.random.randint(2, size=(100, 5))
-        col = np.random.randint(5)
+       col = np.random.randint(5)
```

Comment:
I fixed it, but nothing happened. I think the problem is that pull request is still from martinmozina:master branch, but should be from fix-auc branch. At least that's what we tried to achieve with @astaric.